tfjs-backend-webgpu/src/flags_webgpu.ts

```diff
@@ -82,3 +82,8 @@ ENV.registerFlag('WEBGPU_THRESHOLD_TO_INCREASE_WORKGROUPS_FOR_MATMUL', () => 0);
  * Whether we will run im2col as a separate shader for convolution.
  */
 ENV.registerFlag('WEBGPU_CONV_SEPARATE_IM2COL_SHADER', () => false);
+
+/**
+ * Whether print shader.
+ */
+ENV.registerFlag('WEBGPU_PRINT_SHADER', () => false);
```

Comment:
Do you want to wait for the string flag to be landed?

Reply:
Yes, this change depends on  https://github.com/tensorflow/tfjs/pull/7509